src/reducers/index.test.ts

```diff
@@ -192,4 +193,42 @@ it('updatePhoneNumberToContact', () => {
   expect(reduce(initialState, action)).toEqual(expectedState);
 });
 
+it('updateUserIdsAndPhoneNumbers', () => {
+  const initialState = storeStateDefault;
+  let { userIdsAndPhoneNumbers, ...rest } = initialState;
+
+  let newUserIdsAndPhoneNumbers: UserIdsAndPhoneNumbers = {
+    phoneNumberToUserId: {},
+    userIdToPhoneNumber: {}
+  };
+  newUserIdsAndPhoneNumbers['phoneNumberToUserId'] = {
```

Comment:
initialize everything in one go, i.e., 

let newUserIdsAndPhoneNumbers: UserIdsAndPhoneNumbers = {
    phoneNumberToUserId: {x: 'y', ...